plugins/destiny.py

```diff
@@ -89,14 +89,22 @@ def prepare_lore_cache():
 
     global LORE_CACHE
     LORE_CACHE = {}
+    ghost_count = 0
+    grim_tally = 0
     for level1 in lore_base:
-        for level2 in level1['pageCollection']:
-            for card in level2['cardCollection']:
+        for level2 in level1.get('pageCollection', []):
+            for card in level2.get('cardCollection', []):
                 LORE_CACHE[card['cardName']] = {
                     'cardIntro': card.get('cardIntro', ''),
                     'cardDescription': card['cardDescription'],
                     'cardId': card['cardId']
                 }
+                if 'Ghost Fragment' in card['cardName']:
```

Comment:
Not all Ghost cards have "Ghost Fragment" in their name. For example, the "Osiris: Vision 81" card.